MetaMask/metamask-mobile
# Include L1 fee in the Send flow and on the tx detail page for Optimism

## Description
This PR:
- Includes Optimism's L1 fee in the Send flow, transaction detail page, dapp approval and tx confirmation modals and in all quotes in Swaps (not only in the selected one).

TODOs:
- Once QA is finished, I will merge new code into the swaps-controller, publish it on npm and use it here in package.json. Currently it's pointing to a GitHub commit in the swaps-controller repo for testing purposes.

## Testing
- Switch to the Optimism network
- See correct fees in the Send flow, transaction detail page and in Swaps (including a list of quotes). Try to submit a tx and compare estimated gas fee vs the real one.
- For dapp testing, open e.g. Uniswap in MetaMask app's browser section and try to do a swap with a token that needs an approval and with another one that doesn't. MetaMask's modal windows for approval and tx confirmation will show up with correct estimations. If an approval is needed, an estimated gas fee is higher for a few cents (at this time).
